website/docs/Software/OpenBCISoftware/NeuroFly_Toolkit.md

```diff
@@ -67,4 +64,15 @@ Follow the [Networking Tutorial](../../Software/OpenBCISoftware/02_GUI_Widget_Gu
 
 ![UDP Networking Widget Screenshot](../../assets/TutorialImages/UDP_drone.png)
 
+The EMG Joystick UDP stream will be sent to the IP address and port specified in the Networking Widget. The JSON packets will look like this:
+<!-- #### Packet Example -->
+```json
+{"type":"emgJoystick","data":[0.664,-0.749]}
+```
+Here are the EMG packet values we receive in our Neurofly Unity application:
```

Comment:
I would rephrase this to something like "Here is an example of how we use the deserialized JSON messages in our NeuroFly Unity application."